source/faq.txt

```diff
@@ -106,6 +106,12 @@ closes only inactive sockets, so you cannot interrupt or terminate
 any ongoing operations by using this method. The driver closes these
 sockets only when the process completes.
 
+.. note::
```

Comment:
S: I think you should clip the last few sentences from the preceding paragraph and combine them into this note. Right now, the paragraph says 

> Calling MongoClient.close() closes only inactive sockets, so you cannot interrupt or terminate any ongoing operations by using this method. The driver closes these sockets only when the process completes.

But then this note contradicts that point. Maybe the combined content could express the following points:
- Calling MongoClient.close() closes only inactive sockets
- It also closes existing sessions and txns
- So, even though you might not impact operations by closing a socket, you might impact an operation occurring in a transaction